File: Iniciante/1043.py
linha = input().split()

A = float(linha[0])
B = float(linha[1])
C = float(linha[2])

if A < (B + C) and B < (A + C) and C < (A + B):
  p = A+B+C
  print("Perimetro = {:.1f}".format(p))
else:
  area = ((A+B)*C)/2
  print("Area = {:.1f}".format(area))

File: Iniciante/1043.js
let line1 = lines.shift().split(' ');

let A = Number(line1.shift());
let B = Number(line1.shift());
let C = Number(line1.shift());

if ((A>Math.abs(B-C) && A<(B+C)) || (B>Math.abs(A-C) && B<(A+C)) || (C>Math.abs(A-B) && C<(A-B))) {
  p = A+B+C;
  console.log("Perimetro = " + p.toFixed(1));
} else {
  area = ((A+B)*C)/2;
  console.log("Area = " + area.toFixed(1));
}
